composer.json

```diff
@@ -15,22 +15,21 @@
   "require" : {
     "php": ">=7.4",
     "lexik/jwt-authentication-bundle": "^2.0",
-    "symfony/config": "^3.4|^4.4|^5.2",
-    "symfony/console": "^3.4|^4.4|^5.2",
-    "symfony/dependency-injection": "^3.4|^4.4|^5.2",
+    "symfony/config": "^4.4|^5.3|^6.0",
+    "symfony/console": "^4.4|^5.3|^6.0",
+    "symfony/dependency-injection": "^4.4|^5.3|^6.0",
     "symfony/deprecation-contracts": "^2.1",
-    "symfony/event-dispatcher": "^3.4|^4.4|^5.2",
-    "symfony/http-foundation": "^3.4|^4.4|^5.2",
-    "symfony/http-kernel": "^3.4|^4.4|^5.2",
+    "symfony/event-dispatcher": "^4.4|^5.3|^6.0",
+    "symfony/http-foundation": "^4.4|^5.3|^6.0",
+    "symfony/http-kernel": "^4.4|^5.3|^6.0",
     "symfony/polyfill-php80": "^1.15",
-    "symfony/property-access": "^3.4|^4.4|^5.2",
-    "symfony/security-bundle": "^3.4|^4.0|^5.2",
-    "symfony/security-core": "^3.4|^4.4|^5.2",
-    "symfony/security-guard": "^3.4|^4.4|^5.2",
-    "symfony/security-http": "^3.4|^4.4|^5.2"
+    "symfony/property-access": "^4.4|^5.3|^6.0",
+    "symfony/security-bundle": "^4.4|^5.3|^6.0",
+    "symfony/security-core": "^4.4|^5.3|^6.0",
+    "symfony/security-http": "^4.4|^5.3|^6.0"
   },
   "require-dev": {
-    "akeneo/phpspec-skip-example-extension": "^4.0|^5.0",
+    "babdev/phpspec-skip-example-extension": "^1.0",
```

Comment:
The source package for this looks to be not well maintained and doesn't actually support skipping examples with requirements declared at the example level.  So, this changes to my fork that adds the feature.

Reply:
Thankfully now not needed :)